pubtools/_quay/operator_pusher.py

```diff
@@ -576,6 +576,8 @@ def _create_item_groups_for_version(
                     tag_part,
                     item.origin.split("-")[1].replace(":", "-"),
                 )
+                if is_prerelease:
```

Comment:
Is there a need to split it into two conditions? Can't this all be done in the second condition?